extras/asr-services/providers/vibevoice/transcriber.py

```
@@ -0,0 +1,393 @@
+
+            # Convert to our expected format
+            segments = []
+            for seg in segments_raw:
+                segments.append({
+                    "text": seg.get("Content", ""),
+                    "start": float(seg.get("Start", 0.0)),
+                    "end": float(seg.get("End", 0.0)),
+                    "speaker": f"Speaker {seg.get('Speaker', 0)}",
+                })
+
+            return {"raw_text": raw_output, "segments": segments}
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)